### DISCUSSION TITLE
Discharge to Maxmimise export

### DISCUSSION THREAD

**Discussion by joelegg** [WillCodeForCats/solaredge-modbus-multi#690] [General]
hi  all, 

I'm simply trying to force my battyer (currently at 100%) to discharge to grid using these settings But i cannot get it to do that. 

It charged fine under max self consumption today. I've not been able to get it to dischhage on demand tho. 

![image](https://github.com/user-attachments/assets/dd2d1ed5-6732-4d23-b407-65b8cd240c57)

what could I be doing wrong? 

Thank you

**Comment by joelegg** [comment]
i have changed this so export is not limited to and it still doesn't work (site control limit disabled)

**Reply by zh17home** [reply]
I got Reactive Power Mode : RRCR

**Reply by joelegg** [reply]
Oh is that the settiig i should enable to export to grid from battery ?

**Comment by laba0072** [comment]
HI joelegg,
can you change all values shown on the advance power control panel? 
I'm not able to make changes even if advance power control is enabled.
How did you set Storage Discharge Limit to 11400w ? 

My home battery show these values:
![image](https://github.com/user-attachments/assets/6efca527-6a18-4a06-a98b-b88530bb3331)
I would like to change the Max Discharge Power & Peak Discharge Power.
I think that Storage Discharge Limit  and Max Discharge Power is same...?

kind regards,
laba0072

**Reply by joelegg** [reply]
i dont actually touch these at all as they dont seem to do anything - the battery manages this itself it seems. i'm not entirely sure sorry

**Reply by WillCodeForCats** [reply]
Those are nameplate (info) values from the battery. They are read only.

**Comment by laba0072** [comment]
The values of my screenshot above are read only values.

The document “Technical Note – Power Control Protocol for
SolarEdge Inverters” shows it.

The value “Remote Control Command Discharge Limit” is a R/W value. 
 Need to find a way to set this to 3000w

**Reply by WillCodeForCats** [reply]
Look for entity like `number.solaredge_i1_storage_charge_limit` and `number.solaredge_i1_storage_discharge_limit` where `iX` is the inverter ID number.

**Reply by laba0072** [reply]
Thanks.
I installed the v3.1.0-pre.3. Now I can change storage_discharge_limit by simply unsing the dashboard.
![image](https://github.com/user-attachments/assets/4f0f4c29-e792-4aa2-a056-c228a0691a4b)
Now set to 3000w.

Thanks Seth for coding all this.
